docs/docs/topics/mutual-auth.md

```diff
@@ -0,0 +1,390 @@
+---
+title: Mutual Authentication
+lang: en-US
+sidebarDepth: 1
+meta:
+  - name: keywords
+    content: pomerium identity-access-proxy mutual authentication jwt jwks mtls
+description: >-
+  This page describes the concept of mutual authentication and why it's important.
+---
+
+# Mutual Authentication: A Component of Zero-Trust
+
+Pomerium provides a good layer of security out of the box, but it's not (and can't be) configured for complete [zero trust] right out of the box. This page explains several methods of achieving mutual authentication — a big part of the zero-trust model — with practical examples.
+
+This is a nuanced topic that dives into several specific security practices that provide mutual authentication. You can use the table of contents below to narrow down to the specific tools you're interested in or read the entire doc for a deeper understanding of how these tools work together to support strong infrastructure security.
+
+[[toc]]
+
+## What Is Mutual Authentication?
+
+The concept of mutual authentication is deceptively simple. It means that both sides of a connection can validate the identity of the other. The application of this concept, however, can be varied and complex. Pomerium provides the features and capability to provide mutual authentication between itself and end users, as well as between itself and upstream services, but configuring those external endpoints can vary depending on each service's features.
+
+## Pomerium Defaults
+
+> - **Security**: Moderate
+> - **Difficulty**: Easy
+> - **Encrypted**: Yes (from the end user to Pomerium. It's up to the service to provide a TLS endpoint for Pomerium to use.)
+> - **Mutual Authentication**: None
+
+Let's look at a basic installation of Pomerium on a local network, with a single upstream service. This service contains sensitive data that we want to ensure is confidential and cannot be tampered with.
+
+::: tip Note
+Throughout this guide we will diagram Pomerium as a single service, as it is in all-in-one mode. This is the version provided by our [binaries] and in our Docker-based [Quick Start] page.
+:::
+
+```mermaid
+flowchart LR
+  C(End User)
+  D(Identity Provider)
+  C==o A
+  A-.-D
+  subgraph lan[Internal Network]
+    style lan stroke-dasharray: 5 5
+    A(Pomerium)
+    B(Upstream Service)
+    A --- B
+  end
+```
+
+1. When the user connects to the route `service.example.com` the traffic is received by Pomerium.
+1. Pomerium redirects the user to the identity provider (**IdP**) to sign in and validate their identity.
+1. After signing in to the IdP, Pomerium verifies that the authenticated (**authn**) user is authorized (**authz**) to access the service, and begins communication with it.
```

Comment:
```suggestion
1. After signing in to the IdP, Pomerium verifies that the authenticated (**AuthN**) user is authorized (**AuthZ**) to access the service, and begins communication with it.
```

More of a convention but I typically see these mixed cased. There may be other instances in this docs but I'm just flagging the first.

Reply:
https://en.wikipedia.org/wiki/Web_API_security

> Along with the ease of API integrations come the difficulties of ensuring proper authentication (AuthN) and authorization (AuthZ).